spec/rspec/core/formatters/exception_presenter_spec.rb

```diff
@@ -94,6 +94,21 @@ module RSpec::Core
         EOS
       end
 
+      it 'allows the caller to add encoded description' do
+        the_presenter = Formatters::ExceptionPresenter.new(exception, example,
+                                                           :description => "ジ".encode("CP932"))
```

Comment:
This spec isn't going to work on 1.8.7, check what we do for encoded string in rspec-support and mimic how thats tested? (this is just a suggestion of where to look!)

Reply:
Good catch! 

I found this `if String.method_defined?(:encoding)` https://github.com/rspec/rspec-support/blob/master/spec/rspec/support/differ_spec.rb#L81

For the moment I'm not able to install ruby-1.8.7-p374 on my machine. It fails with rvm so I will use the CI. Sorry for the noise.